graph-ts/chain/tendermint.proto

```diff
@@ -0,0 +1 @@
+fig/tendermint/codec/v1/codec.proto
```

Comment:
The `tendermint.ts` file will be generated based on the `fig/tendermint/codec/v1/codec.proto` file. I would remove it from here and keep only the files that will be processed with `as-proto`.